Hippo.Core/Domain/Payment.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Hippo.Core.Domain
+{
+    public class Payment
+    {
+        [Key]
+        public int Id { get; set; }
+        public string SlothId { get; set; }
+        public string TrackingNumber { get; set; } // KFS tracking number
+        public DateTime CreatedOn { get; set; } = DateTime.UtcNow;
+        public decimal Amount { get; set; }
+        public string Status { get; set; }
+
+        public string ClusterChartString { get; set; }
+        public string OrderChartStrings { get; set; } //.Serialize() of the billings chart strings and percentages used
```

Comment:
I think you should remove both of these.  Just replace with a single string `Description` field and dump payment details in there.  Like `$5 paid.  $4 from CS1, $1 from CS2, sent to CS3`.  No need to json or anything